### ISSUE TITLE
COG Doesn't Load all Configs from Resources

### ISSUE THREAD

**Issue by Draco18s** [lawremi/CustomOreGen#68]
So I was updating my dev source from v18 to v20 so I could play around with the new block adjacency code.

But I get this error:

    [12:23:25] [Client thread/ERROR] [CustomOreGen]: Problem loading world config
    CustomOreGen.Config.ParserException: CustomOreGen Config Error at Element <Import> [line 397]: No files found matching 'modules/default/AppliedEnergistics.xml'.
    	at CustomOreGen.Config.ValidatorImport.validateChildren(ValidatorImport.java:40) ~[ValidatorImport.class:?]
    	at CustomOreGen.Config.ValidatorNode.validate(ValidatorNode.java:75) ~[ValidatorNode.class:?]
    	at CustomOreGen.Config.ValidatorNode.validateChildren(ValidatorNode.java:110) ~[ValidatorNode.class:?]
    	at CustomOreGen.Config.ValidatorRoot.validateChildren(ValidatorRoot.java:20) ~[ValidatorRoot.class:?]
    	at CustomOreGen.Config.ValidatorNode.validate(ValidatorNode.java:75) ~[ValidatorNode.class:?]
    	at CustomOreGen.Config.ValidatorNode.validateChildren(ValidatorNode.java:110) ~[ValidatorNode.class:?]
    	at CustomOreGen.Config.ValidatorNode.validate(ValidatorNode.java:75) ~[ValidatorNode.class:?]
    	at CustomOreGen.Config.ConfigParser.parseFile(ConfigParser.java:135) ~[ConfigParser.class:?]
    	at CustomOreGen.Server.WorldConfig.<init>(WorldConfig.java:197) ~[WorldConfig.class:?]
    	at CustomOreGen.Server.WorldConfig.<init>(WorldConfig.java:75) ~[WorldConfig.class:?]
    	at CustomOreGen.Server.GuiCustomOreGenSettings.initGui(GuiCustomOreGenSettings.java:65) [GuiCustomOreGenSettings.class:?]
    	at net.minecraft.client.gui.GuiScreen.setWorldAndResolution(GuiScreen.java:294) [GuiScreen.class:?]
    	at net.minecraft.client.Minecraft.displayGuiScreen(Minecraft.java:854) [Minecraft.class:?]
    	at CustomOreGen.Server.GuiCustomOreGenSettings$GuiOpenMenuButton.mousePressed(GuiCustomOreGenSettings.java:770) [GuiCustomOreGenSettings$GuiOpenMenuButton.class:?]
    	at net.minecraft.client.gui.GuiScreen.mouseClicked(GuiScreen.java:245) [GuiScreen.class:?]
    	at net.minecraft.client.gui.GuiCreateWorld.mouseClicked(GuiCreateWorld.java:392) [GuiCreateWorld.class:?]
    	at net.minecraft.client.gui.GuiScreen.handleMouseInput(GuiScreen.java:344) [GuiScreen.class:?]
    	at net.minecraft.client.gui.GuiScreen.handleInput(GuiScreen.java:313) [GuiScreen.class:?]
    	at net.minecraft.client.Minecraft.runTick(Minecraft.java:1720) [Minecraft.class:?]
    	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1028) [Minecraft.class:?]
    	at net.minecraft.client.Minecraft.run(Minecraft.java:951) [Minecraft.class:?]
    	at net.minecraft.client.main.Main.main(Main.java:164) [Main.class:?]
    	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0_67]
    	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[?:1.7.0_67]
    	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.7.0_67]
    	at java.lang.reflect.Method.invoke(Method.java:606) ~[?:1.7.0_67]
    	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.11.jar:?]
    	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.11.jar:?]
    	at net.minecraftforge.gradle.GradleStartCommon.launch(GradleStartCommon.java:78) [start/:?]
    	at GradleStart.main(GradleStart.java:45) [start/:?]

I've unzipped the new source code and the new config xml files and deleted the /eclipse/config/CustomOreGen folder, so that it would do a full refresh.  Yet for some reason COG is not extracting about a third of its default modules: only 20 out of 36.

**Comment by reteo** [comment]
Odd... I just downloaded v20, and it still has the original 20 configs in both the "modules/default/" folder, as well as in the "CustomOreGen_Config_Default.xml" file.   The updated configuration files were added on July 31st, and as far as I know, is not in any of the releases yet.

I'm gonna download the COG directly from the master, just to make sure I didn't do something stupidly boneheaded during the pull requests.
